types/react-big-calendar/index.d.ts

```diff
@@ -303,6 +303,7 @@ export interface EventWrapperProps<TEvent extends object = Event> {
     label: string;
     continuesEarlier: boolean;
     continuesLater: boolean;
+    children: React.ReactNode;
```

Comment:
This does not cover both use cases: component and function. The function modus is not just a functional component.